bpf/kmesh/workload/sockops.c

```diff
@@ -257,7 +269,9 @@ int sockops_prog(struct bpf_sock_ops *skops)
     case BPF_SOCK_OPS_ACTIVE_ESTABLISHED_CB:
         if (!is_managed_by_kmesh(skops))
             break;
-        observe_on_connect_established(skops->sk, OUTBOUND);
+        if (enable_metric()) {
+            observe_on_connect_established(skops->sk, OUTBOUND);
```

Comment:
`enable_metric` should be called in `observe_on_connect_established`

Reply:
Can you tell me why should do it?

Reply:
Done